Flaky test `TestCopyBuffer`

Test `TestCopyBuffer` seems to be flaky: [CI attempt #1](https://github.com/ethersphere/bee/actions/runs/3211181773/jobs/5249171328)

```
--- FAIL: TestCopyBuffer (0.00s)
   buffer_test.go:221: timeout
   ```

--- a/pkg/file/buffer_test.go
+++ b/pkg/file/buffer_test.go
@@ -12,7 +12,6 @@ import (
 	"math/rand"
 	"strconv"
 	"testing"
-	"time"
 
 	"github.com/ethersphere/bee/pkg/file"
 	"github.com/ethersphere/bee/pkg/swarm"
@@ -86,7 +85,6 @@ func TestChunkPipe(t *testing.T) {
 			// receive the writes
 			// err may or may not be EOF, depending on whether writes end on
 			// chunk boundary
-			timer := time.NewTimer(time.Second)
 			readTotal := 0
 			for {
 				select {
@@ -101,8 +99,6 @@ func TestChunkPipe(t *testing.T) {
 							t.Fatal(err)
 						}
 					}
-				case <-timer.C:
-					t.Fatal("timeout")
 				}
 			}
 		})
@@ -193,7 +189,6 @@ func TestCopyBuffer(t *testing.T) {
 			// err may or may not be EOF, depending on whether writes end on
 			// chunk boundary
 			expected := dataSize
-			timer := time.NewTimer(time.Second)
 			readTotal := 0
 			readData := []byte{}
 			for {
@@ -217,8 +212,6 @@ func TestCopyBuffer(t *testing.T) {
 							t.Fatal(err)
 						}
 					}
-				case <-timer.C:
-					t.Fatal("timeout")
 				}
 			}
 		})

